microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -5,147 +5,223 @@ author: MSEdgeTeam
 
-#### 1.1.8 Altering Browser Settings
+
+###### 1.1.8 Altering browser settings
 
 Your extension must not, without appropriate user consent, alter, or appear to alter, browser functionality or settings including, but not limited to: the address bar search provider and suggestions, the start or home page, the new tab page, and adding or removing favorite.
 
-Any alteration to the browser settings should be explicitly documented in the description of your extension.
+Any changes made to the device settings must be done with the user's knowledge and consent and must be easily reversible by the user.
+
+Any alteration of browser settings such as new tabs and search engines should be mentioned in the listing or description, and they should use standard APIs to achieve that functionality.  They must not override default settings without user permission.
+
+
+<!-- ------------------------------ -->
+#### 1.2 Security
+
+Extensions must declare all permissions required for their functionality in the manifest.  Your Extension must only request those permissions that are essential for functioning.
 
-Your extension may only revise key settings to replace a Microsoft webpage or service with that of a third-party (such as require use of a third-party search engine or set the home page to a third-party web property) if you are employed by or otherwise associated with such third-party.
+Declarative Net Request rules must be clearly declared within the extension and should not be imported remotely unless necessary for functionality.
 
-### 1.2 Security
+Extensions must not collect or track sensitive user information without clear consent and must clearly state any user data handling in their privacy policy and user sensitive data must be encrypted with proper mechanisms like SSL and not transported via insecure channels.
 
-Your extension must not jeopardize or compromise user security, or the security or functionality of the device, system or related systems.
+Extensions must not promote deceptive installation of other extensions.
 
-#### 1.2.1 Content Security Policies
+Bulk submissions of extensions with the same functionality and code are not allowed.
+
+Extensions should not exploit security vulnerabilities of other extensions or applications and must keep authentication information secure.
+
+Extensions that function as crypto wallets can only be published by verified company accounts
+
+In general, your extension must not jeopardize or compromise user security, or the security or functionality of the device, system or related systems.
+
+
+###### 1.2.1 Content security policies
 
 If you make any changes to your extension beyond the described functionality, any changes to code must be compliant with the [Microsoft Edge content security policy](csp.md#relaxing-the-default-policy).  For example, your extension should not download a remote script and subsequently run that script in a manner that is not consistent with the described functionality.
```

Comment:
Link text needs to match destination title/heading.  Linking to Relaxing section seems mistake.  Changed to:
```
If you make any changes to your extension beyond the described functionality, 
any changes to code must be compliant with 
the Content Security Policy for Microsoft Edge extensions; 
see [Content Security Policy (CSP)](./csp.md).  
For example, your extension should not download a remote script and subsequently 
run that script in a manner that is not consistent with the described functionality.
```